src/Dapr.Client/DaprClient.cs

```diff
@@ -143,6 +143,16 @@ public abstract class DaprClient
         /// <returns>A <see cref="ValueTask{T}" /> that will return the value when the operation has completed.</returns>
         public abstract ValueTask<TValue> GetStateAsync<TValue>(string storeName, string key, ConsistencyMode? consistencyMode = default, CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Gets a list of values associated with the <paramref name="keys" /> from the Dapr state store.
+        /// </summary>
+        /// <param name="storeName">The name of state store to read from.</param>
+        /// <param name="keys">The list of keys to get values for.</param>
+        /// <param name="parallelism">The number of concurrent get operations the Dapr runtime will issue to the state store.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
+        /// <returns>A <see cref="ValueTask{GetBulkItem}" /> that will return the list of values when the operation has completed.</returns>
```

Comment:
```suggestion
        /// <returns>A <see cref="ValueTask{IEnumerable{GetBulkItem}}" /> that will return the list of values when the operation has completed.</returns>
```

Reply:
This throws a compile error: `Type parameter declaration must be an identifier not a type`

Reply:
hmmm. something is wrong tho - the return type *isn't* `ValueTask{GetBulkItem}`. it should reflect `ValueTask{IReadOnlyList{GetBulkItem}}`

Reply:
I changed it to ValueTask{IReadOnlyList}, it seems like the compiler doesn't support more than one level of generics for cref.